modules/database/sqlite/activate.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Actions to run when the module gets activated.
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Copies the db.php file in wp-content and reloads the page.
+ *
+ * @since n.e.x.t
+ */
+return function() {
+	// Bail early if the SQLite3 class does not exist.
+	if ( ! class_exists( 'SQLite3' ) ) {
+		return;
+	}
+
+	$destination = WP_CONTENT_DIR . '/db.php';
+
+	// Place database drop-in if not present yet, except in case there is
+	// another database drop-in present already.
+	if ( ! defined( 'PERFLAB_SQLITE_DB_DROPIN_VERSION' ) && ! file_exists( $destination ) ) {
+		// Init the filesystem to allow copying the file.
+		global $wp_filesystem;
+		if ( ! $wp_filesystem ) {
+			require_once ABSPATH . '/wp-admin/includes/file.php';
+			WP_Filesystem();
+		}
+
+		// Copy the file, replacing contents as needed.
+		if ( $wp_filesystem->touch( $destination ) ) {
+
+			// Get the db.copy file contents, replace placeholders and write it to the destination.
+			$file_contents = str_replace(
+				'{SQLITE_IMPLEMENTATION_FOLDER_PATH}',
+				__DIR__,
+				file_get_contents( __DIR__ . '/db.copy' )
+			);
+
+			$wp_filesystem->put_contents( $destination, $file_contents );
+		}
+	}
+
+	// As an extra safety check, bail if the current user cannot update
+	// (or install) WordPress core.
+	if ( ! current_user_can( 'update_core' ) ) {
+		return;
+	}
+
+	// Otherwise install WordPress in the SQLite database before the redirect,
+	// so that the user does not end up on the WordPress installation screen
+	// but rather remains in the Performance Lab modules screen.
+	add_filter(
+		'wp_redirect',
+		function( $redirect_location ) {
+			// If the SQLite DB already exists, simply ensure the module is
+			// active there.
+			require_once __DIR__ . '/constants.php';
```

Comment:
The change you made in https://github.com/WordPress/performance/pull/547/commits/eb77632076cba62ade5a3b286a60fb645864ab9b around the default value for the `DATABASE_TYPE` constant causes a fatal error when activating the module. Please make sure to test that activation of the module still works when making sensitive changes like this :)

The problem is that here we load the database code "on-the-fly", and the drop-in isn't loaded. This then causes the `wp-includes/sqlite/db.php` file to return early and not load any of the necessary classes. So we need to force the constant to be the correct value here too in order to fix that fatal error.
```suggestion
		function( $redirect_location ) {
			if ( ! defined( 'DATABASE_TYPE' ) ) {
				define( 'DATABASE_TYPE', 'sqlite' );
			}
			require_once __DIR__ . '/constants.php';

			// If the SQLite DB already exists, simply ensure the module is
			// active there.
```